lib/rubocop/cop/rspec/factory_bot/consistent_parentheses_style.rb

```diff
@@ -0,0 +1,105 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      module FactoryBot
+        # Use a consistent style for parentheses in factory bot calls
+        #
+        # @example
+        #
+        #   # bad
+        #   create :user
+        #   build(:user)
+        #   create(:login)
+        #   create :login
+        #
+        # @example `EnforcedStyle: require_parentheses`
+        #
+        #   # good
+        #   create(:user)
+        #   create(:user)
+        #   create(:login)
+        #   build(:login)
+        #
+        # @example `EnforcedStyle: omit_parentheses`
+        #
+        #   # good
+        #   create :user
+        #   build :user
+        #   create :login
+        #   create :login
+        #
+        class ConsistentParenthesesStyle < Base
+          extend AutoCorrector
+          include ConfigurableEnforcedStyle
+          include RuboCop::Cop::Util
+
+          def self.autocorrect_incompatible_with
+            [Style::MethodCallWithArgsParentheses]
+          end
+
+          MSG_ENFORCE_PARENS = 'Prefer method call with parentheses'
+          MSG_OMIT_PARENS = 'Prefer method call without parentheses'
+
+          FACTORY_CALLS = Set[:create, :build, :create_list,
+                              :build_list, :create_pair,
+                              :build_stubbed_list, :attributes_for,
+                              :attributes_for_list, :attributes_for_pair,
+                              :build_stubbed_pair, :build_stubbed,
+                              :build_pair ].freeze
+
+          # @!method factory_call(node)
+          def_node_matcher :factory_call, <<-PATTERN
+              (send
+                ${(const nil? {:FactoryGirl :FactoryBot}) nil?} %FACTORY_CALLS
```

Comment:
Cosmetic: we have a helper for this in `lib/rubocop/rspec/factory_bot/language.rb` if you like it.
Could be just
```
${#factory_bot? nil?} %FACTORY_CALLS
```
As a free benefit, it also covers `::FactoryBot` calls.